tests/nextflow.config

```diff
@@ -4,9 +4,46 @@
 ========================================================================================
 */
 
-// TODO nf-core: Specify any additional parameters here
-// Or any resources requirements
-params.modules_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/'
-params.pipelines_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/refs/heads/abotyper'
+params {
+    modules_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/'
+    pipelines_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/'
 
-aws.client.anonymous = true // fixes S3 access issues on self-hosted runners
+    // Default output directory for tests
+    outdir = 'test_results'
+}
+
+// Resource limits for all tests
+process {
+    resourceLimits = [
+        cpus: 2,
+        memory: '3.GB',
+        time: '2.h'
+    ]
+}
+
+// Enable Docker for all tests
+docker {
+    enabled = true
+    // Remove Unix-specific user mapping for Windows compatibility
+    // Add network timeout settings
+    registry = 'docker.io'
```

Comment:
that shouldn't be there, the default registry should really be quay.io.
For which modules do you need that?